docgen/src/examples/tourism/App.js

```diff
@@ -1,7 +1,8 @@
-/* eslint react/prop-types: 0 */
+/* eslint react/prop-types: 0, import/no-extraneous-dependencies: 0 */
```

Comment:
why is this added? :)

Reply:
Otherwise ESLint complaint about the maps package but not the normal one.